docs/tutorials/faq.md

```diff
@@ -253,6 +253,14 @@ integer durations. Although the HLS spec allows it, Shaka Player requires that
 the duration must be precise with decimal-floating-point or decimal-integer
 number. If this is not your case, please open an issue so we can investigate it.
 
+<hr>
+
+**Q:** My HLS stream uses PlayReady with the embedded license url.
```

Comment:
Can you rephrase this as an actual question?  Does PlayReady with embedded license URL not work if you do it naively?